cmd/svcat/plugin/plugin.go

```diff
@@ -46,6 +46,9 @@ const (
 	// EnvPluginVerbose is the -v=LEVEL flag
 	EnvPluginVerbose = EnvPluginGlobalFlagPrefix + "_V"
 
+	// EnvPluginOutput is the -o=FORMAT flag
```

Comment:
I just tested this in plugin mode and I apologize for sending you down the wrong path. It won't be sent to us as a global flag after all so the changes to this file can be reverted. Our default plugin wiring will handle setting it without this (because it will come in as an env var like this `KUBECTL_PLUGINS_LOCAL_FLAG_OUTPUT=json`).

Reply:
![image](https://user-images.githubusercontent.com/33884483/38955725-b5fd7c4a-4312-11e8-8180-ec6507f37bbe.png)